Vwitter/UserCell.m

```diff
@@ -0,0 +1,102 @@
+//
+//  UserCell.m
+//  Vwitter
+//
+//  Created by Christina Li on 7/11/22.
+//
+
+#import "UserCell.h"
+#import "Follow.h"
+
+@implementation UserCell
+
+- (void)awakeFromNib {
+    [super awakeFromNib];
+    // Initialization code
+}
+
+- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
+    [super setSelected:selected animated:animated];
+
+    // Configure the view for the selected state
+}
+
+- (void)setUser:(PFUser *)user{
+    _user = user;
+
+    [self refreshData];
+
+}
+
+- (void)refreshData {
+
+    NSString *at = @"@";
+    self.username.text = [NSString stringWithFormat:@"%@%@", at, self.user.username];
+    
+    if ([self.user.objectId isEqualToString:[PFUser currentUser].objectId]) {
+        self.followStatusButton.hidden = YES;
+    }
+    
+    [PFCloud callFunctionInBackground:@"existsFollow"
+                       withParameters:@{@"currentUserObjectId":[PFUser currentUser].objectId, @"followingUserObjectId":self.user.objectId}
+                                block:^(id exists, NSError *error) {
+      if (!error) {
+          NSLog (@"%d", ((NSNumber *)exists).boolValue);
+          if (((NSNumber *)exists).boolValue) {
+              [self.followStatusButton setTitle:@"Following" forState:UIControlStateNormal];
+          }
+          else {
+              [self.followStatusButton setTitle:@"Follow" forState:UIControlStateNormal];
+          }
+      }
+    }];
```

Comment:
important: We did discuss already that this whole view (and the view controllers that create it) should be refactored so that the entire view controller just needs to make 1 single network request to fetch all of the data needed to render the initial screen. 

You can then just create new model classes to hold this info, which would usually be known as "SomethingViewModel". In this case, your new model class would be something like `UserCellViewModel`, it can be a subclass of `NSObject`, and just has three simple properties, an `NSString` "username" property, a `BOOL` "isFollowing" property, and a `NSString` "userId" property.  The view controller will make the network request to get the list of users, and then, using that data, it will build up the array of these new `UserCellViewModel` objects and pass them to the cell so it can render.